packages/terra-tabs/src/clinical-lowlight-theme/HiddenTab.module.scss

```diff
@@ -0,0 +1,25 @@
+:local {
+  .clinical-lowlight-theme {
+    --terra-tabs-workspace-hidden-tab-hidden-background-color: #222a2e;
```

Comment:
```suggestion
    --terra-common-tabs-hidden-tab-hidden-background-color: #222a2e;
```
Should we rename all the variables to use common tabs instead of workspace

Reply:
These variables need to be prefixed with "--terra-tabs-" since the package name is still "terra-tabs". Causes lint violation upon change